lib/utils.ts

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import attempt from 'lodash/attempt';
```

Comment:
I'm not keen on this pattern of abstracting the lodash functions to this file, as it means if we stop using one for some reason it's much hard to spot. 
If instead, you import the function you need where you need it, if you stop using the function it will be picked up as unused by the linter and can be removed.
In general, I'm highly suspicious of files named "utils" or "helpers", since they may as well be called `stuff.ts` and tend to accumulate cruft and dead code.

Reply:
Fixed in latest commit